introduction/de/01-introduction.asciidoc

```diff
@@ -0,0 +1,12 @@
+== Einführung
+
+Diese Weiterbildungsserie bietet eine Einführung in InnerSource. 
+InnerSource ist die Anwendung von Open-Source Praktiken und -Prinzipien auf die Softwareentwicklung im Unternehmen. 
+Die InnerSource-Software bleibt dabei Eigentum des Unternehmens, kann jedoch von jedem Mitarbeiter des Unternehmens genutzt, verbessert und erweitert werden. 
```

Comment:
```suggestion
Die InnerSource-Software bleibt dabei Eigentum des Unternehmens. Jeder Mitarbeiter kann sie nutzen und Verbesserungen oder Erweiterungen beitragen.
```

Reply:
I'd like to avoid an impression that InnerSource creates a "free for all" where everybody gets direct write access.